packages/@aws-cdk/aws-ec2/test/integ.vpc-endpoint.lit.ts

```diff
@@ -0,0 +1,36 @@
+import iam = require('@aws-cdk/aws-iam');
+import cdk = require('@aws-cdk/cdk');
+import ec2 = require('../lib');
+
+const app = new cdk.App();
+const stack = new cdk.Stack(app, 'aws-cdk-ec2-vpc-endpoint');
+
+/// !show
```

Comment:
Can we put this in a subclass of stack so that first argument is always `this` instead of `stack` (which is non-idiomatic). I know we have many examples like that, but we should start somewhere...